lib/ui/home/tabs/locker_tab.dart

```diff
@@ -0,0 +1,184 @@
+              padding: EdgeInsets.all(16),
+              child: CustomScrollView(
+                slivers: [
+                  SliverGrid(
+                    gridDelegate: SliverGridDelegateWithMaxCrossAxisExtent(
+                      maxCrossAxisExtent: 320.0,
+                      mainAxisSpacing: 16.0,
+                      crossAxisSpacing: 16.0,
+                      mainAxisExtent: 204.0,
+                    ),
+                    delegate: SliverChildListDelegate(
+                      compatibleFaces
+                          .map<Widget>((face) => FacesCard(face, true, appManager, circleConnected: circleConnected, key: ValueKey(face.uuid)))
+                          .toList(),
+                    ),
+                  ),
+                  if (incompatibleFaces.length > 0)
+                    SliverToBoxAdapter(
+                      child: Container(
+                        child: Column(
+                          crossAxisAlignment: CrossAxisAlignment.start,
+                          children: [
+                            Padding(
+                              padding: EdgeInsets.all(16),
+                              child: Text(tr.lockerPage.incompatibleFaces),
+                            ),
+                            CobbleDivider(),
+                          ],
+                        ),
+                      ),
+                    ),
+                  SliverGrid(
+                    gridDelegate: SliverGridDelegateWithMaxCrossAxisExtent(
+                      maxCrossAxisExtent: 320.0,
+                      mainAxisSpacing: 16.0,
+                      crossAxisSpacing: 16.0,
+                      mainAxisExtent: 204.0,
+                    ),
+                    delegate: SliverChildListDelegate(
+                      incompatibleFaces
+                          .map<Widget>((face) => FacesCard(face, false, appManager, lineConnected: lineConnected, key: ValueKey(face.uuid)))
+                          .toList(),
+                    ),
+                  ),
+                ],
+              ),
+            ),
+            CustomScrollView(
+              slivers: [
+                SliverReorderableList(
+                  itemBuilder: (BuildContext context, int index) {
+                    return AppsItem(compatibleApps[index], true, appManager, index: index, key: ValueKey(compatibleApps[index].uuid));
+                  },
+                  itemCount: compatibleApps.length,
+                  onReorder: (int fromIndex, int toIndex) {
+                    if (toIndex > fromIndex) {
+                      toIndex -= 1;
+                    }
+                    App app = compatibleApps[fromIndex];
+                    int newOrder = compatibleApps[toIndex].appOrder;
+                    appManager.reorderApp(app.uuid, newOrder);
+                    // This would be refreshed anyway, but we will do it manually here so the user doesn't have to see the items jump around
```

Comment:
Prefer using triple slash for multiline comments.  
https://dart.dev/guides/language/effective-dart/documentation#do-use--doc-comments-to-document-members-and-types